### PR TITLE
Updated ChangeLog for 6.0

### PR DESCRIPTION
**Pull request by PaulWessel**
Based on git log entry lines and memory.  Please add items I may have missed (not everything has an entry in the log - but those who added a feature will remember better), and organization and sorting will be improved.  Do we have good enough introduction to modern mode elsewhere?

### CODE REVIEW

File: doc/rst/source/changes.rst [hunk 1]
```diff
@@ -4,6 +4,218 @@
...
+The entire cookbook, tutorial and gallery examples all use modern mode. In modern mode,
+the default graphics format is PDF and scripts can open up the plots in the default
+viewer automatically.
+
+New modules
+-----------
+
+Apart from modern mode we have added a few modules that are accessible to all users:
+
+#. :doc:`psevents` makes a snapshot of all time-dependent events.
```
Comment (seisman) [inline doc/rst/source/changes.rst]:
```suggestion
#. :doc:`events` makes a snapshot of all time-dependent events.
```

File: doc/rst/source/changes.rst [hunk 2]
```diff
@@ -4,6 +4,218 @@
...
+   respectively.
+
+Module enhancements
+-------------------
+
+Several modules have obtained new options to extend their capabilities:
+
+*  :doc:`grdfilter` now accepts the **-r** option to set grid node registration.
+
+*  :doc:`psclip` has a new option **-W**\ *pen* to draw the clip path as well as
```
Comment (seisman) [inline doc/rst/source/changes.rst]:
```suggestion
*  :doc:`clip` has a new option **-W**\ *pen* to draw the clip path as well as
```

File: doc/rst/source/changes.rst [hunk 3]
```diff
@@ -4,6 +4,218 @@
...
+   golden ratio, as well as **NODE** and **NODEP** operators, and added more
+   OpenMP support for operators **LDISTG**, **PSI**, **TCRIT**, **PLM**, and **PLMg**.
+
+*  :doc:`rose` can now take **-JX** instead of **-S** so all plot modules take **-J**.
+
+*  :doc:`grdedit` can now take **-J** and add meta-data to the grid header.
+
+*  :doc:`gmt2kml takes new option **-E** to extract altitudes stored in the Extended data property.
+
+*  :doc:`/supplements/seis/pspolar` and :doc:`/supplements/seis/meca` can let beachball size scale
```
Comment (seisman) [inline doc/rst/source/changes.rst]:
```suggestion
*  :doc:`/supplements/seis/polar` and :doc:`/supplements/seis/meca` can let beachball size scale
```

**Comment by PaulWessel** [comment]
I think this one is good enough for RC5, @seisman or @joa-quim

**Pull request merged**